Directory.Build.props

```diff
@@ -27,6 +27,7 @@
   <PropertyGroup>
     <NetFrameworkMinimum>net462</NetFrameworkMinimum>
     <UwpMinimum>uap10.0.16299</UwpMinimum>
+    <UwpNet9Minimum>net8.0-windows10.0.26100.0</UwpNet9Minimum>
```

Comment:
Property name mentions "net9" but value is "net8"

Reply:
Fixed! 🙂